app/src/main/java/chat/rocket/android/contacts/adapter/ContactRecyclerViewAdapter.kt

```diff
@@ -0,0 +1,97 @@
+package chat.rocket.android.contacts.adapter
+
+import android.content.Intent
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import chat.rocket.android.R
+import chat.rocket.android.chatrooms.adapter.*
+import chat.rocket.android.main.ui.MainActivity
+import chat.rocket.android.util.extensions.inflate
+import kotlin.collections.HashMap
+
+
+class ContactRecyclerViewAdapter(
+        private val context: MainActivity,
+        private val contactArrayList: List<ItemHolder<*>>,
+        private val contactHashMap: HashMap<String, String>
```

Comment:
Is this attribute being used anywhere in this file?  contactHashMap?  If not can we get rid of it?

Reply:
ok